<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Multiplexed Metagenomic Deep Sequencing To Analyze the Composition of High-Priority Pathogen Reagents</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	
    <!-- Customized Bootstrap + Font Awesome + Solarized -->
    <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/favicon.png"/>	

	<!-- Typekit -->
	<script type="text/javascript" src="//use.typekit.net/xeu8jut.js"></script>
	<script type="text/javascript">try{Typekit.load();}catch(e){}</script>
		
	<!-- Google Analytics -->	
   <script>
     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
   
     ga('create', 'UA-55282509-1', 'auto');
     ga('send', 'pageview');
   
   </script>
			
</head>

<body>
	
	<div id="header">
		<nav class="navbar navbar-default navbar-static-top" role="navigation">	 
			<div class="container">	
				
				<div class="navbar-header">
					<object class="pull-left logo" data="/images/logo.svg" type="image/svg+xml"></object>
					<a class="navbar-brand" href="/">Stenglein Lab</a>
				</div>
				
				<ul class="nav navbar-nav navbar-right">
					
					<li>
					
					<a href="/blog/">blog</a>
					</li>
					
					<li>
					
					<a href="/protocols/">protocols</a></li>
					
					<li>
					
					<a href="/software/">software</a></li>
					
					<li class="active">
					
					<a href="/papers/">papers</a></li>
               <!--
					
					<li>
					
					<a href="/projects/">projects</a></li>	  
               -->
					
					<li>
					
					<a href="/team/">team</a></li>	  	  
				</ul>
			</div>
		</nav>	
	</div>
	
	<div class="container">	
		
	<div class="row">
	<div class="col-md-3">
		<div class="media">
			
			<!-- <img class="pull-left pad-right media-object image-fluid" src="/images/papers/dual_indexing.jpg"> -->
			<img class="pull-left media-object image-fluid" src="/images/papers/dual_indexing.jpg">
			
	   </div>
	</div>
	<div class="col-md-8">
			<div class="media-body titlebox">
				<!-- <div class="title media-heading"> -->
				<div class="title media-heading">
					Multiplexed Metagenomic Deep Sequencing To Analyze the Composition of High-Priority Pathogen Reagents
				</div>
				<p>
				<div class="smallhead">
					Wilson MR, Fedewa G, Stenglein MD, Olejnik J, Rennick LJ, Nambulli S, Feldmann F, Duprex WP, Connor JH, MÃ¼hlberger E, DeRisi JL
					2016.
					mSystems
				</div>
			</div>
		</div>
	</div>
	<div class="col-md-1>
	</div
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-md-3">
		<div class="bigspacer"></div>
		<div class="glyphbox note">
			
			<div class="smallhead">
				PDF
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-file-text-o fa-fw"></i>
				<a class="off" href="/pdfs/papers/Wilson_2016_mSystems.pdf">Wilson_2016_mSystems.pdf</a>
			</div>
			<div class="bigspacer"></div>
			
			
			
			<div class="smallhead">
				DOI
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-external-link fa-fw"></i>
				<a class="off" href="http://dx.doi.org/10.1128/mSystems.00058-16">10.1128/mSystems.00058-16</a>
			</div>
			<div class="bigspacer"></div>
			
			
		</div>
	</div>
	<div class="col-md-8">
		<div class="post">
			<h2 id="abstract">Abstract</h2>

<p>Laboratories studying high-priority pathogens need comprehensive methods to confirm microbial species and strains while also detecting contamination. Metagenomic deep sequencing (MDS) inventories nucleic acids present in laboratory stocks, providing an unbiased assessment of pathogen identity, the extent of genomic variation, and the presence of contaminants. Double-stranded cDNA MDS libraries were constructed from RNA extracted from in vitro-passaged stocks of six viruses (La Crosse virus, Ebola virus, canine distemper virus, measles virus, human respiratory syncytial virus, and vesicular stomatitis virus). Each library was dual indexed and pooled for sequencing. A custom bioinformatics pipeline determined the organisms present in each sample in a blinded fashion. Single nucleotide variant (SNV) analysis identified viral isolates. We confirmed that (i) each sample contained the expected microbe, (ii) dual indexing of the samples minimized false assignments of individual sequences, (iii) multiple viral and bacterial contaminants were present, and (iv) SNV analysis of the viral genomes allowed precise identification of the viral isolates. MDS can be multiplexed to allow simultaneous and unbiased interrogation of mixed microbial cultures and (i) confirm pathogen identity, (ii) characterize the extent of genomic variation, (iii) confirm the cell line used for virus propagation, and (iv) assess for contaminating microbes. These assessments ensure the true composition of these high-priority reagents and generate a comprehensive database of microbial genomes studied in each facility. MDS can serve as an integral part of a pathogen-tracking program which in turn will enhance sample security and increase experimental rigor and precision.</p>

<h2 id="importance">Importance</h2>

<p>Both the integrity and reproducibility of experiments using select agents depend in large part on unbiased validation to ensure the correct identity and purity of the species in question. Metagenomic deep sequencing (MDS) provides the required level of validation by allowing for an unbiased and comprehensive assessment of all the microbes in a laboratory stock.</p>


		</div>
	</div>
	<div class="col-md-1"></div>
</div>

	
	</div>
		
	<div id="footer"><span style="display:none">foo</span></div>
	    		
	<!-- MathJax support -->
	<!-- <script type="text/javascript" -->
	 <!-- <src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script> -->
	<!-- MathJax config -->
	<!-- <script src="/js/mathjax.config.js"></script>	 -->
			
</body>
</html>

