<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"/><title>Virus identification in unknown tropical febrile illness cases using deep sequencing.</title><meta name="viewport" content="width=device-width, initial-scale=1.0"><link href="/css/style.css" rel="stylesheet" media="screen"><link rel="shortcut icon" href="/images/favicon.png"/> <script src="//use.typekit.net/xeu8jut.js"></script><script>try{Typekit.load()}catch(e){}</script> <script>!function(e,a,t,n,c,o,s){e.GoogleAnalyticsObject=c,e[c]=e[c]||function(){(e[c].q=e[c].q||[]).push(arguments)},e[c].l=1*new Date,o=a.createElement(t),s=a.getElementsByTagName(t)[0],o.async=1,o.src=n,s.parentNode.insertBefore(o,s)}(window,document,"script","//www.google-analytics.com/analytics.js","ga"),ga("create","UA-55282509-1","auto"),ga("send","pageview")</script></head><body><div id="header"> <nav class="navbar navbar-default navbar-static-top" role="navigation"><div class="container"><div class="navbar-header"> <object class="pull-left logo" data="/images/logo.svg" type="image/svg+xml"></object> <a class="navbar-brand" href="/">Stenglein Lab</a></div><ul class="nav navbar-nav navbar-right"><li> <a href="/blog/">blog</a></li><li class="active"> <a href="/papers/">papers</a></li><li> <a href="/team/">team</a></li></ul></div> </nav></div><div class="container"><div class="bigspacer"></div><div class="row"><div class="col-md-12"><div class="media"> <img class="pull-left pad-right media-object" src="/images/papers/filtering.png"><div class="media-body"><div class="title media-heading"> Virus identification in unknown tropical febrile illness cases using deep sequencing.</div><p><div class="smallhead"> Yozwiak NL*, Skewes-Cox P*, Stenglein MD*, Balmaseda A, Harris E, DeRisi JL 2012. PLoS neglected tropical diseases</div></div></div></div><div class="col-md-2"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-3"><div class="bigspacer"></div><div class="glyphbox note"><div class="smallhead"> PDF</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-file-text-o fa-fw"></i> <a class="off" href="/pdfs/papers/Yozwiak_2012_PLoS_NTD.pdf">Yozwiak_2012_PLoS_NTD.pdf</a></div><div class="bigspacer"></div><div class="smallhead"> DOI</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-external-link fa-fw"></i> <a class="off" href="http://dx.doi.org/10.1371/journal.pntd.0001485">10.1371/journal.pntd.0001485</a></div><div class="bigspacer"></div><div class="smallhead"> Share</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-twitter fa-fw"></i> <a class="off" href="http://twitter.com/share?url=http://stenglein-lab.github.io/papers/Yozwiak-seq_tropical_febrile/&text=Virus%20identification%20in%20unknown%20tropical%20febrile%20illness%20cases%20using%20deep%20sequencing." target="_blank">tweet</a></div></div></div><div class="col-md-8"><div class="post"><h1>Abstract</h1><p>Dengue virus is an emerging infectious agent that infects an estimated 50-100 million people annually worldwide, yet current diagnostic practices cannot detect an etiologic pathogen in [?]40% of dengue-like illnesses. Metagenomic approaches to pathogen detection, such as viral microarrays and deep sequencing, are promising tools to address emerging and non-diagnosable disease challenges. In this study, we used the Virochip microarray and deep sequencing to characterize the spectrum of viruses present in human sera from 123 Nicaraguan patients presenting with dengue-like symptoms but testing negative for dengue virus. We utilized a barcoding strategy to simultaneously deep sequence multiple serum specimens, generating on average over 1 million reads per sample. We then implemented a stepwise bioinformatic filtering pipeline to remove the majority of human and low-quality sequences to improve the speed and accuracy of subsequent unbiased database searches. By deep sequencing, we were able to detect virus sequence in 37% (45/123) of previously negative cases. These included 13 cases with Human Herpesvirus 6 sequences. Other samples contained sequences with similarity to sequences from viruses in the Herpesviridae, Flaviviridae, Circoviridae, Anelloviridae, Asfarviridae, and Parvoviridae families. In some cases, the putative viral sequences were virtually identical to known viruses, and in others they diverged, suggesting that they may derive from novel viruses. These results demonstrate the utility of unbiased metagenomic approaches in the detection of known and divergent viruses in the study of tropical febrile illness.</p></div></div><div class="col-md-1"></div></div></div><div id="footer"><span style="display:none">foo</span></div> <script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script> <script src="/js/mathjax.config.js"></script></body></html>